src/app/pages/RuntimeTransactionDetailPage/index.tsx

```diff
@@ -201,6 +221,60 @@ export const RuntimeTransactionDetailView: FC<{
             </>
           )}
 
+          <dt>{t('common.recentTransactions')}</dt>
+          <dd>
+            <Box sx={{ display: 'block', overflowX: 'auto' }}>
+              {transfers?.map((transfer, i) => {
+                const params = transfer.evm_log_params
+                if (!params) return null
+
+                const from = params.find(p => p.name === 'from')?.value as string | undefined
+                const to = params.find(p => p.name === 'to')?.value as string | undefined
+                const amount = params.find(p => p.name === 'value')?.value as string | undefined
+                const symbol = transfer.evm_token?.symbol
+
+                return (
+                  <Box
+                    key={i}
+                    sx={{
+                      display: 'flex',
+                      alignItems: 'center',
+                      gap: 2,
+                      mb: 2,
+                      minWidth: '100%',
+                      whiteSpace: 'nowrap',
+                    }}
+                  >
+                    <TokenTypeTag tokenType={transfer.evm_token?.type} />
+                    <Typography variant="body2">{t('common.from')} </Typography>
+                    {from ? <AccountLink scope={scope} address={from} alwaysTrim /> : '?'}
+
+                    <Typography variant="body2">{t('common.to')}</Typography>
+                    {to ? <AccountLink scope={scope} address={to} alwaysTrim /> : '?'}
+
+                    <Typography variant="body2">
+                      {t('common.for')} {amount} {symbol}
```

Comment:
This doesn't handle NFTs well
https://pr-1939.oasis-explorer.pages.dev/mainnet/sapphire/tx/0x739566b66acb5626a50f7bc1833f5a372d645145b1547a58fb5d67b9ae9ff35a
![image](https://github.com/user-attachments/assets/b5fb24b8-61ee-43c6-b0d2-0956d94612ba)

should be more like https://explorer.dev.oasis.io/mainnet/sapphire/address/0x998633BDF6eE32A9CcA6c9A247F428596e8e65d8/token-transfers#transfers
![image](https://github.com/user-attachments/assets/e2017594-6c4d-4a5e-a084-ae2be6698c4d)

Reply:
Example transaction that transfers both types
https://pr-1939.oasis-explorer.pages.dev/mainnet/sapphire/tx/0xc0c9511c5233447830d1769876d3fa16b4f6a64c89309e775ad1c3cb5e96992c